usr/share/rear/skel/BAREOS/usr/lib/systemd/system/multi-user.target.wants/bareos-fd.service

```diff
@@ -0,0 +1 @@
+../bareos-fd.service
```

Comment:
We often double check for running agents and start them if needed, for example in
https://github.com/rear/rear/blob/24680de5be94f55ecf8a40a406e8ee13e9a95502/usr/share/rear/verify/PPDM/default/390_check_for_running_ppdm_agent_service.sh#L4-L10